map.html

```diff
@@ -69,11 +69,15 @@
     <div class="topbarSection"><span class="totalProgressPercent">0</span>%</div> 
 </div>
 <div class="mapContainer" id="mapContainer">
+	<h1 id="title">Map</h1>
```

Comment:
Do we still need the title map on here if the majority of the page is the map? 
it seemed self-evident enough(to me) to not have the title.

Reply:
All the other pages have it, so I figured why not. But you are right that it's not really necessary.